test/intern.js

```diff
@@ -12,6 +12,16 @@ define({
     host: 'localhost',
     port: 4444
   },
+  kibana: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 5620
+  },
+  elasticsearch: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 9220
```

Comment:
It may make sense to put the defaults into a new module and support overriding them with environment variables, then read that module wherever we use the config (similar to how we do the [babel options](https://github.com/elastic/kibana/blob/master/src/optimize/babelOptions.js)).

test/functional/config.js
```js
exports.kibanaPort = parseInt(process.env.TEST_UI_ES_PORT, 10) || 9220;
exports.esPort = parseInt(process.env.TEST_UI_ES_PORT, 10) || 9221;
```

everywhere else
```js
elasticsearch: {
  port: require('requirefrom')('test')('functional/config').esPort;
}
```